pytab/core.py

```diff
@@ -300,6 +307,32 @@ def cli(
     system_info = hwi.get_system_info()
     click.echo(" success!")
 
+    # Logic for GPU Selection
+    gpus = system_info["gpu"]["gpu_elements"]
```

Comment:
There is no key "gpu_elements" in the Hardware Information.
The pattern is:
```json
{
    "gpu": [
        {
            "id": "GPU1",
        },
        {
            "id": "GPU2",
        }
    ]
}
```

So `system_info["gpu"]` returns a List of individual GPU Elements already.